README.md

```diff
@@ -11,14 +11,14 @@
 
 ## Instructions
 
-- Download Tor browser from the official site ([link](https://www.torproject.org/download/download))
+- Download Tor using the following instructions: ([link](https://2019.www.torproject.org/docs/debian.html.en))
```

Comment:
Well, since about 95% or so of hidden services run on Linux, it would be more wise to use that. People should use Linux for a hidden service anyway due to anonimity and stuff. Either way, it's your repository, but I just included some Tor Project documentation.